model-archiver/model_archiver/tests/unit_tests/test_model_packaging.py

```diff
@@ -52,11 +53,11 @@ def test_gen_model_archive(self, patches):
 
     def test_export_model_method(self, patches):
 
-        patches.export_utils.check_mar_already_exists.return_value = '/Users/ghaipiyu/'
-        patches.export_utils.check_custom_model_types.return_value = '/Users/ghaipiyu', ['a.txt', 'b.txt']
+        patches.export_utils.check_mar_already_exists.return_value = '/Users/dummyUser/'
+        patches.export_utils.check_custom_model_types.return_value = '/Users/dummyUser', ['a.txt', 'b.txt']
         patches.export_utils.zip.return_value = None
 
         package_model(self.args, ModelExportUtils.generate_manifest_json(self.args))
         patches.export_utils.validate_inputs.assert_called()
-        patches.export_utils.zip.assert_called()
+        patches.export_utils.archive.assert_called()
```

Comment:
@vdantu Can you add a test to check the code path with .tgz as the extension ? 
Right now this test checks the "default" code path (which now has branches in it due to multiple formats)

Reply:
I hadn't added the test because these tests don't seem to be testing the actual output. But you do have a valid point. I am going to add similar test for now. Will put something in the backlog to update some of the tests.